src/libPMacc/include/memory/buffers/HostDeviceBuffer.hpp

```diff
@@ -36,6 +36,8 @@ namespace PMacc {
         typedef HostBufferIntern<T_Type, T_dim> HostBufferType;
         typedef DeviceBufferIntern<T_Type, T_dim> DeviceBufferType;
     public:
+        typedef HostBuffer<T_Type, T_dim> HostBuffer;
```

Comment:
Because `HostBufferIntern` is the implementation and `HostBuffer` is the "interface". Only the latter should be exposed to the user.